apps/dashboard/app/apps/nav_bar.rb

```diff
@@ -166,4 +144,43 @@ def initialize(nav_item, partial_path)
       @links = nav_item.links.flatten.compact
     end
   end
+
+  def self.set_multikey_value(hash: [], keys: [], value: nil)
+    keys.each do |key|
+      hash[key.to_sym] = value
+    end
+  end
+
+  def self.static_templates
+    @@static_templates ||= {}.tap do |hash|
```

Comment:
```suggestion
    @static_templates ||= {}.tap do |hash|
```

I think we want a class instance variable, not a class variable. Rubocop would tell you the same. 

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ClassVars